packages/EasyApiToken/src/External/AwsCognito/JwkFetcher.php

```diff
@@ -93,7 +74,7 @@ private function fetchKeysFromAws(UserPoolConfigInterface $userPoolConfig): arra
         $keys = [];
 
         foreach ($response['keys'] as $jwk) {
-            $keys[$jwk['kid']] = $this->convertJwkToPem($jwk);
+            $keys[(string)$jwk['kid']] = new Key($this->convertJwkToPem($jwk), $jwk['alg'] ?? 'HS256');
```

Comment:
The same here, the `\EonX\EasyApiToken\Interfaces\AlgorithmsInterface::HS256` const please.

Reply:
Refactored.